PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+<!--
+Thank you for your pull request. Please provide a description above and review
+the requirements below.
+
+Bug fixes and new features should include tests and possibly benchmarks.
+
+Contributors guide: https://github.com/bcoin-org/bcoin/blob/master/.github/CONTRIBUTING.md
+-->
+
+##### Checklist
+<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->
```

Comment:
Should we also say that code coverage is not decreased or is this too arbitrary to enforce?

Reply:
Too arbitrary, in same cases just removing code can decrease coverage, because that code was more covered that other parts. There are other cases.